portality/scripts/mark_notifications_seen.py

```diff
@@ -0,0 +1,21 @@
+# ~~ MarkNotificationsSeen:Script ~~
+from portality.lib import dates
+from portality.bll import DOAJ
+
+
+def mark_notifications_seen(until):
+    # ~~-> Notifications:Service ~~
+    notifications_svc = DOAJ.notificationsService()
+    notifications_svc.mark_all_as_seen(until=until)
+
+
+if __name__ == "__main__":
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-u", "--until", help="mark all notifications before this as seen")
```

Comment:
Do we need to move the help string to constants?

Reply:
It's a thought, but I don't think it's so important for the command line scripts as they are only for us.  I'm going to leave it where it is, unless there's a strong urge to move it.